expo-maps/ios/GoogleMapsView.swift

```diff
@@ -0,0 +1,24 @@
+import GoogleMaps
+
+public final class GoogleMapsView: UIView {
+  internal var mapView: UIView
+
+  init() {
+    // just for now we do authentication here
+    // should be moved to module's function
+    GMSServices.provideAPIKey("AIzaSyDbgaRNTr3PhYdj_PL7jY_o9u3R08Gf8Ao")
```

Comment:
We should provide an empty string as API key in the repository

Reply:
I think this can be hardcoded for now to be more convenient, as @tsapeta mentioned in his comments above. We won’t be billed for this key, in worst case scenario it will just get blocked after exceeding the free limit of requests.